rockcraft.yaml

```diff
@@ -74,15 +56,29 @@ parts:
       run-server.sh: app/k8s/run-server.sh
       k8s-init.sh: app/k8s/k8s-init.sh
       k8s-bootstrap.sh: app/k8s/k8s-bootstrap.sh
+      rock-requirements.txt: requirements/rock.txt
+    stage:
+      - app/k8s/run-server.sh
+      - app/k8s/k8s-init.sh
+      - app/k8s/k8s-bootstrap.sh
+      - requirements/rock.txt
+    permissions:
+      - path: app/k8s
+        owner: 584792
```

Comment:
For other reviewers, these ID values come from the shared user referred to in the first file.